JavaScript/JavaScriptSDK/Sender.ts

```diff
@@ -168,22 +180,17 @@ module Microsoft.ApplicationInsights {
                 // Send data only if disableTelemetry is false
                 if (!this._config.disableTelemetry()) {
```

Comment:
I'm not sure if I understand. We step into the `if` statement if telemetry is enabled (not disabled).